src/OrgasmControl.cpp

```diff
@@ -34,26 +34,30 @@ namespace OrgasmControl {
     void updateMotorSpeed() {
       // Motor increment goes 0 - 100 in ramp_time_s, in steps of 1/update_fequency
       float motor_increment = (
-          (float)Config.motor_max_speed /
+          (float)Config.motor_max_speed - (float)Config.motor_start_speed  /
```

Comment:
I'm not sure if the order of operations is preserved here or if the compiler will divide first then do the subtraction. Parenthesis for clarity.

```suggestion
          (float)(Config.motor_max_speed - Config.motor_start_speed)  /
```

Reply:
Agreed, best it wrap it to be sure.